runtime/README.md

```diff
@@ -381,69 +378,21 @@ DPE will first collect measurements and concatenate them in a byte buffer
 To derive a CDI for a given context, DPE shall use KeyVault hardware with the
 following inputs:
 
-If `FIPS_MODE` is disabled for the DPE command OR Sign(SYMMETRIC=true), DPE will
-
 * CDI = Runtime Firmware CDI (from KeyVault)
 * Label = LABEL parameter provided to Sign or CertifyKey
 * Context = `MEASUREMENT_DATA`
 
-If `FIPS_MODE` is enabled for the DPE command AND Sign(SYMMETRIC=false), DPE will
-
-* CDI = Output from TRNG
-* Label = LABEL parameter provided to Sign or CertifyKey
-* Context = `MEASUREMENT_DATA`
-
-In either case, the CDI shall be loaded into KeyVault slot 0.
+The CDI shall be loaded into KeyVault slot 0.
 
 ### Leaf Key Derivation
 
-To derive an asymmetric key for Sign and CertifyKey
-
-If `FIPS_MODE` is disabled for the DPE command OR Sign(SYMMETRIC=true), DPE will
+To derive an asymmetric key for Sign and CertifyKey, RT will
 
 * Derive an ECC P384 keypair from KV slot 0 CDI into KV slot 1
 * For CertifyKey: Request the public key
 * For Sign: Sign passed data
 * Erase KeyVault slots 0 and 1
 
-If `FIPS_MODE` is enabled for the DPE command AND Sign(SYMMETRIC=false), DPE will
-
-* If the key has been derived before and is cached in KeyVault
-  * Use cached key from KeyVault
-* If key is not cached
-  * Derive an ECC P384 keypair from KV slot 0 CDI into the next available Runtime
-    KeyVault Slot
-  * If no slots are available, evict the oldest Runtime slot and use that slot.
-* For CertifyKey: Request the public key
-* For Sign: Sign passed data
-* Erase KeyVault slot 0
-
-### FIPS Mode Caching
-
-Caliptra has 32 KeyVault slots and a max of 24 DPE contexts. To accomodate
```

Comment:
Is that part FIPS specific? I thought regardless of FIPS we still have those upper limits?

Reply:
I guess this be derived implicitly from max TCI nodes so maybe add a statement in that section that max contexts == max TCI nodes

Reply:
In the deterministic (regular) mode, we can derive a key, use it, and then clear the slot. So number of hw KeyVault slots isn't a limitation.